src/vm/eventpipefile.cpp

```diff
@@ -189,6 +205,8 @@ void EventPipeFile::WriteEvent(EventPipeEventInstance &instance, ULONGLONG captu
     }
     CONTRACTL_END;
 
+    ASSERT(m_isInitialized);
```

Comment:
nit: use `PRECONDITION`

Reply:
Opted to use `ASSERT`s inside `#ifdef DEBUG` since this `BOOL` is _only_ meant for debug time sanity checks and that seems to indicate that intent.